Datadog/TargetSupport/DatadogIntegrationTests/DatadogCrashReportingIntegrationTests.xctestplan

```diff
@@ -33,21 +33,9 @@
   },
   "testTargets" : [
     {
-      "skippedTests" : [
-        "IntegrationTests",
-        "LoggingScenarioTests",
-        "RUMManualInstrumentationScenarioTests",
-        "RUMMobileVitalsScenarioTests",
-        "RUMModalViewsScenarioTests",
-        "RUMNavigationControllerScenarioTests",
-        "RUMResourcesScenarioTests",
-        "RUMScrubbingScenarioTests",
-        "RUMSwiftUIScenarioTests",
-        "RUMTabBarControllerScenarioTests",
-        "RUMTapActionScenarioTests",
-        "TracingManualInstrumentationScenarioTests",
-        "TracingURLSessionScenarioTests",
-        "TrackingConsentScenarioTests"
+      "selectedTests" : [
+        "CrashReportingWithLoggingScenarioTests\/testCrashReportingCollectOrSendWithLoggingScenario()",
+        "CrashReportingWithRUMScenarioTests\/testCrashReportingCollectOrSendWithRUMScenario()"
```

Comment:
This change is on purpose. By default, all new integration tests are added to both XCTest plans (_integration_ and _crash reporting_), meaning we will run it twice on CI. To avoid this, I disabled "add new tests automatically" in _crash reporting_ XCTest plan. Apparently, this means changing from opt-out to opt-in in `.xctestplan` 👍.